### PR TITLE
add german language

### PR DESCRIPTION
**Pull request by ellerbrock**
g3rm4n

### CODE REVIEW

File: languages/de.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+categories: Kategorien
+search: Suche
+tags: Tags
+tagcloud: Tag Cloud
+tweets: Tweets
+prev: zurück
+next: weiter
+comment: Kommentare
+archive_a: Archiv
```
Comment (Xuanwo) [inline languages/de.yml]: Is this a typo?

Comment (ellerbrock) [inline languages/de.yml]: Hi Xuanwo,

what do you mean? Where do you think is an typo?

Comment (Xuanwo) [inline languages/de.yml]: `Archive` instead of `Archiv`?
Sorry, I'm not familiar with German, maybe I confused something.

Comment (ellerbrock) [inline languages/de.yml]: Ah ok i got you.
No thats correct. Some german words are the same or very similar sometimes to english.
Some words we keep the speaking in the original language. Like Tweet or Tag Cloud, if you translate them to german no tech guy would know what u mean :)

**Comment by ellerbrock** [comment]
oh another thing i forgot to write you.
you have some broken links and mixed content warnings in your template demo site:

url: https://hexo.io/hexo-theme-landscape/

when you open the developer tools u can see which files are effected.

here a more detailed log:

```
'results' attribute is deprecated and will be removed in M53, around September 2016. See https://www.chromestatus.com/features/5738199536107520 for more details.
hexo.io/:93 Mixed Content: The page at 'https://hexo.io/hexo-theme-landscape/' was loaded over HTTPS, but requested an insecure image 'http://placehold.it/350x150.jpg'. This content should also be served over HTTPS.
hexo.io/:241 Mixed Content: The page at 'https://hexo.io/hexo-theme-landscape/' was loaded over HTTPS, but requested an insecure image 'http://placehold.it/350x150.jpg'. This content should also be served over HTTPS.
hexo.io/:91 GET https://hexo.io/assets/wallpaper-2311325.jpg 404 ()
hexo.io/:92 GET https://hexo.io/assets/wallpaper-878514.jpg 404 ()
hexo.io/:90 GET https://hexo.io/assets/wallpaper-2572384.jpg 404 ()
```

cheers

**Comment by Xuanwo** [comment]
Well, I got you.
Let merge this pr first and create a new issue to slove the problem.

**Pull request merged**